Makefile

```diff
@@ -56,12 +56,12 @@ setup-pyenv:
 	./scripts/pyenv_setup.sh
 
 ensure-venv:
-	@./scripts/ensure-venv.sh
+	./scripts/ensure-venv.sh
```

Comment:
Trying to make calling them more consistent (`@` vs `bash` vs straight call).

Reply:
What is the difference?

Reply:
The difference (IIUC) is that one will print the command itself in the output (with `@`).